packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -398,7 +398,7 @@ Future<Null> diagnoseXcodeBuildFailure(
   }
```

Comment:
do you want to clean up lines 394/395 while you're here? they're the ones that looked weird to me...

Reply:
Done in #14478